style.css

```diff
@@ -63,14 +61,22 @@ body {
     font-family: 'Roboto', sans-serif;
     font-size: 55px;
     flex-grow:1;
-    padding: 0px 0px 130px 150px;
+    padding-left: 150px;
 }
 
 .photo {
     flex-grow: 1; 
-    padding: 0px 0px 200px 0px; 
+    padding-bottom: 65px;
```

Comment:
for now I'd like you to practise the shorthand notation for margin and padding
`padding: top right bottom left;`
so padding bottom of 65px is:
`padding: 0 0 65px;` _notice the last 0 missed it's copied from the second value_

Reply:
got it :)